main.tf

```diff
@@ -81,6 +81,23 @@ resource "openstack_images_image_v2" "fedora-34-20210427" {
   }
 }
 
+resource "openstack_images_image_v2" "fedora-35-20211102" {
+  for_each = toset(var.architectures)
+
+  region           = var.region
+
+  name             = "Fedora 35 (${each.value}) [2021-11-02]"
+  visibility       = "public"
+  image_source_url = "https://mirrors.kernel.org/fedora/releases/34/Cloud/${each.value}/images/Fedora-Cloud-Base-35-1.2.${each.value}.qcow2"
```

Comment:
Shouldn't this be 
 > image_source_url = "https://mirrors.kernel.org/fedora/releases/35/Cloud/${each.value}/images/Fedora-Cloud-Base-35-1.2.${each.value}.qcow2"

Instead of 
> image_source_url = "https://mirrors.kernel.org/fedora/releases/34/Cloud/${each.value}/images/Fedora-Cloud-Base-35-1.2.${each.value}.qcow2"

Reply:
Excellent eye, @Odilhao. PR made here: https://github.com/vexxhost/terraform-openstack-images/pull/4